<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCommandStartBattle</class>
 <widget class="QDialog" name="DialogCommandStartBattle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Start a battle...</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_9">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-139</y>
          <width>454</width>
          <height>703</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QGroupBox" name="groupBoxTroopID">
             <property name="title">
              <string>Troop ID</string>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,1">
                  <item>
                   <widget class="QRadioButton" name="radioButtonVariableConstant">
                    <property name="text">
                     <string>ID:</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="PanelPrimitiveValue" name="widgetVariableConstant" native="true">
                    <property name="enabled">
                     <bool>true</bool>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>300</width>
                      <height>27</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>300</width>
                      <height>27</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_16">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>68</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
                  <item>
                   <widget class="QRadioButton" name="radioButtonRandom">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="text">
                     <string>Random (in map property)</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_17">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>158</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBoxBattleMap">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="title">
              <string>Battle map</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="0" column="0" rowspan="2" colspan="2">
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="0" colspan="2">
                 <widget class="QRadioButton" name="radioButtonID">
                  <property name="text">
                   <string>ID:</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2" colspan="2">
                 <widget class="PanelPrimitiveValue" name="panelPrimitiveValueIDBattleMap" native="true">
                  <property name="minimumSize">
                   <size>
                    <width>300</width>
                    <height>27</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>300</width>
                    <height>27</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="0" column="4">
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="1" column="0">
                 <widget class="QRadioButton" name="radioButtonSelect">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1" rowspan="2" colspan="4">
                 <widget class="PanelSelectPosition" name="panelSelectPosition" native="true">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>286</width>
                    <height>111</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>111</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <spacer name="verticalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>98</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="3" column="0">
                 <widget class="QRadioButton" name="radioButtonNumber">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1" colspan="2">
                 <widget class="QLabel" name="labelIDMAp">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>ID map:</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="4" rowspan="5">
                 <spacer name="horizontalSpacer_11">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>38</width>
                    <height>158</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="4" column="0" rowspan="4">
                 <spacer name="verticalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>128</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="5" column="1">
                 <widget class="QLabel" name="labelY">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Y:</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="3">
                 <widget class="PanelPrimitiveValue" name="widgetY" native="true">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>250</width>
                    <height>27</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>250</width>
                    <height>27</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="6" column="1" colspan="2">
                 <widget class="QLabel" name="labelYPlus">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Y plus:</string>
                  </property>
                 </widget>
                </item>
                <item row="6" column="3">
                 <widget class="PanelPrimitiveValue" name="widgetYplus" native="true">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>250</width>
                    <height>27</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>250</width>
                    <height>27</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="7" column="1">
                 <widget class="QLabel" name="labelZ">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Z:</string>
                  </property>
                 </widget>
                </item>
                <item row="7" column="3">
                 <widget class="PanelPrimitiveValue" name="widgetZ" native="true">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>250</width>
                    <height>27</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>250</width>
                    <height>27</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="3" column="3">
                 <widget class="PanelPrimitiveValue" name="widgetIdMap" native="true">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>250</width>
                    <height>27</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>250</width>
                    <height>27</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="4" column="3">
                 <widget class="PanelPrimitiveValue" name="widgetX" native="true">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>250</width>
                    <height>27</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>250</width>
                    <height>27</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="QLabel" name="labelX">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>X:</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBoxOptions">
             <property name="title">
              <string>Options</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_3">
                <item row="0" column="0">
                 <widget class="QCheckBox" name="checkBoxEscape">
                  <property name="text">
                   <string>Allow escape</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1" rowspan="2">
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>68</width>
                    <height>48</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="1" column="0">
                 <widget class="QCheckBox" name="checkBoxGameOver">
                  <property name="text">
                   <string>Defeat causes Game Over</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBoxTransition">
             <property name="title">
              <string>Transition</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_5">
                <item row="0" column="0">
                 <widget class="QLabel" name="labelStart">
                  <property name="text">
                   <string>Start:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QComboBox" name="comboBoxTransitionStart">
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="currentIndex">
                   <number>-1</number>
                  </property>
                 </widget>
                </item>
                <item row="0" column="3" rowspan="2">
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>38</width>
                    <height>58</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="labelEnd">
                  <property name="text">
                   <string>End:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QComboBox" name="comboBoxTransitionEnd">
                  <property name="currentIndex">
                   <number>-1</number>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="PanelPrimitiveValue" name="panelPrimitiveValueTransitionColorStart" native="true">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>250</width>
                    <height>27</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>250</width>
                    <height>27</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="PanelPrimitiveValue" name="panelPrimitiveValueTransitionColorEnd" native="true">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>250</width>
                    <height>27</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>250</width>
                    <height>27</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PanelPrimitiveValue</class>
   <extends>QWidget</extends>
   <header>panelprimitivevalue.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PanelSelectPosition</class>
   <extends>QWidget</extends>
   <header>panelselectposition.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogCommandStartBattle</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogCommandStartBattle</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
